ci/jobs/pull_request.yml

```diff
@@ -37,7 +37,7 @@ jobs:
           cp ../../../terraform-config/terraform.tf .
           cp ../../../terraform-config/terraform.tfvars .
           terraform init
-          terraform workspace select management
```

Comment:
PR jobs must be planned against management

Reply:
Reverted back to original